### PR TITLE
Change retrieved_at format from date-time to date

### LINKED ISSUE #37: date should be valid for retrieved_at
@inglesp seems to required a date-time, but sometimes we only have a date (for example if it's the export date of one of many historic files on FTP site)

### PR DESCRIPTION
**Pull request by jpmckinney**
Closes #37

### CODE REVIEW

File: build/filing-schema.json [hunk 1]
```diff
@@ -66,7 +66,7 @@
     "retrieved_at": {
       "description": "Date-time this was retrieved from the source",
```
Comment (sebbacon) [inline build/filing-schema.json]: Description should be updated too...

File: build/gazette-notice-schema.json [hunk 2]
```diff
@@ -432,7 +432,7 @@
     "retrieved_at": {
       "description": "The time at which the source URL was requested",
```
Comment (sebbacon) [inline build/gazette-notice-schema.json]: This description needs updating

**Comment by sebbacon** [comment]
@jpmckinney, can you provide a reference to the conversation where this was agreed please (I guess in flowdock)?

**Comment by jpmckinney** [comment]
#37

**Comment by jpmckinney** [comment]
Fixed descriptions

**Comment by CountCulture** [comment]
Are we testing somewhere that the datetime will also pass validation?

**Comment by jpmckinney** [comment]
Yes, all the old sample datum with retrieved at still have date times except for the one that was changed to a date

**Comment by jpmckinney** [comment]
All comments resolved. Merging.

**Pull request merged**